### ISSUE TITLE
Does not resolve correctly for paths using `oneOf` property

### ISSUE THREAD

**Issue by aychtang** [RangerMauve/json-schema-from-path#2]
We have a requirement to resolve paths for schemas that use oneOf.

Example:
```
const schema = {
  properties: {
    example: {
        oneOf: [{
            title: 'target',
            type: 'string',
            value: 'hello world'
        }]
    }
  }
}

getSchemaFromPath(schema, 'example.target') // => should resolve to {title: 'target', type: 'string',value: 'hello world'}
```

@RangerMauve: I will implement this and make a PR, please let me know if you had any thoughts.

**Comment by aychtang** [comment]
Also we will need to update the README if we add this feature

**Comment by RangerMauve** [comment]
Thanks for the fix! Out in 2.0.0. 😁
